datadog/resource_datadog_synthetics_test_.go

```diff
@@ -2375,6 +2855,66 @@ func buildDatadogSyntheticsBrowserTest(d *schema.ResourceData) *datadogV1.Synthe
 	return syntheticsTest
 }
 
+func buildDatadogSyntheticsMobileTest(d *schema.ResourceData) *datadogV1.SyntheticsMobileTest {
+	syntheticsTest := datadogV1.NewSyntheticsMobileTestWithDefaults()
+	if attr, ok := d.GetOk("message"); ok {
+		syntheticsTest.SetMessage(attr.(string))
+	}
+	// if attr, ok := d.GetOk("monitor_id"); ok { // TODO uncomment and find out why it's unexpected
```

Comment:
?

Reply:
I think i asked you about this to confirm a few days ago, but i thought about it a bit more and the id is generated from the backend so doesn't really make sense to put in in the request 🤔